proto/buf.yaml

```diff
@@ -0,0 +1,19 @@
+# This module represents the "proto" root.
+version: v1
+name: buf.build/crypto-org-chain/chain-main
```

Comment:
this name/url gives 404 -- do we need to register it on buf.build?

Reply:
https://docs.buf.build/bsr/usage#push-a-module

Reply:
ok, for releases, we can perhaps specify an action to do it: https://docs.buf.build/ci-cd/github-actions

Reply:
Created #852